docs/docs-content/enterprise-version/upgrade/upgrade-vmware/airgap.md

```diff
@@ -0,0 +1,186 @@
+    Uncompressing Airgap Pack - aws-alb Version 4.0.17  100%
+    Setting up Packs
+    - Pushing Pack aws-alb:2.5.1
+    Setting up Images
+    Setup Completed
+    ```
+
+5.  Use the following command template to execute the new Palette airgap installation bin.
+
+    ```shell
+    chmod +x airgap-<version>.bin && ./airgap-<version>.bin
+    ```
+
+    Consider the following example for reference.
+
+    ```shell
+    chmod +x airgap-v4.2.13.bin && ./airgap-v4.2.13.bin
+    ```
+
+6.  After the Palette airgap binary is verified and uncompressed, it uploads the updates to your existing Palette
+    instance. This step takes some time to complete.
+
+    Once the airgap binary is executed, you will receive a **Setup Completed** success message.
+
+    ```shell
+    Verifying archive integrity...  100%   MD5 checksums are OK. All good.
+    Uncompressing Airgap Setup - Version 4.2.13  100%
+    Setting up CLI
+    Setting up Manifests
+    Setting up Packs
+    ...
+
+    //highlight-next-line
+    Setup Completed
+    ```
+
+7.  Log in to the Palette system console.
+
+8.  From the left **Main Menu**, select **Enterprise Cluster** and then select the **Profile** tab.
+
+9.  Copy the configurations for your CSI, CNI, and Kubernetes layers to an external location for backup.
+
+    :::warning
+
+    If your Enterprise Cluster profile has configuration changes or additions, make sure to back up all the customized
+    values to preserve them after the upgrade.
+
+    :::
+
+    ![Self-hosted Palette system console with the highlighted Enterprise Cluster profile configuration values.](/enterprise-version_upgrade-upgrade_vmware_non-airgap_copy_configurations.webp)
+
+10. From the left **Main Menu**, select **Administration** > **Pack Registries**. Then, next to your **spectro-packs**
+    registry, select the **three-dot Menu** > **Sync**.
+
+    Palette initiates a registry sync, which can take up to 30 minutes to complete.
+
+11. From the left **Main Menu**, select **Update Management**. Once the registry sync is done, Palette will prompt you
+    to update to the new version. Click **Update**.
+
+    ![Self-hosted Palette system console with Update Management open and the Update button highlighted.](/enterprise-version_upgrade-upgrade_vmware_non-airgap_update.webp)
+
+12. Review the changes that the upgrade introduces. If your Enterprise Cluster profile had any configuration changes or
+    additions, add them to the **Current configuration** pane and select **Confirm**.
+
+    :::warning
+
+    Make sure to transfer the configuration changes and additions in all Enterprise Cluster profile layers to preserve
+    them after the upgrade.
+
+    :::
+
+    ![Self-hosted Palette system console with the upgrade preview pane.](/enterprise-version_upgrade-upgrade_vmware_non-airgap_upgrade-preview.webp)
+
+    The system console locks for a couple of minutes while the upgrade is applied.
```

Comment:
Same as before, we really need to. invest more time to heal the user here and understand what to do. It's very easy to mess up and get this step wrong 😅